ubiquity-fundraising-research-april-may-2024.md

```diff
@@ -0,0 +1,87 @@
+## Introduction
+The goal of the research is to obtain objective information about the perception of Ubiquity in order to develop materials and communications for investors to secure a funding round. During April and May 2024, Sergey found and conducted interviews with 8 crypto influencers and investors who have extensive experience in investing in crypto projects.
+
+To obtain objective feedback and information, the conversations were conducted anonymously, in the format of interviews with experts, not as a pitch to investors. Experts were provided with Ubiquity's public materials and no internal information was provided. Some experts conducted their own research to comprehensively assess Ubiquity.
+
+The trigger for starting the research was Sergey's interaction in March-April 2024 with an agency that supported ChainGPT. Previously, Sergey believed that he objectively assessed Ubiquity's situation, but it turned out during the interaction with the agency that this might not be the case. According to the agency's assessment in March 2024, Ubiquity had low market potential.
+
+Sergey presumed that investors might come to a similar assessment, which would significantly hinder Ubiquity's ability to obtain the next funding round. Seeing Ubiquity through the eyes of independent expert investors, Alex and Sergey could build investor communication in such a way as to highlight Ubiquity's strengths and mitigate its weaknesses. Observing Alex's approach to finding investors, Sergey believes that this research will significantly save time for Ubiquity in preparing for the next funding round.
```

Comment:
I've been iterating based on feedback from my pitch calls, same as we did for seed.